spec/requests/api/submissions/index_spec.rb

```diff
@@ -31,6 +31,24 @@
       expect(body).to eq []
     end
 
+
+    fit "returns only draft submissions when state is draft" do
```

Comment:
**suggestion**: should `s/fit/it` otherwise all of the specs in this file will be skipped.